src/main/java/com/twilio/oai/resolver/php/PhpParameterResolver.java

```diff
@@ -27,26 +30,26 @@ public void resolveProperties(CodegenParameter codegenParameter) {
             codegenParameter.dataType = "array[]";
         }
         if (codegenParameter.dataType.equalsIgnoreCase(OBJECT) ||
-                codegenParameter.dataType.equals(LanguageConventionResolver.LIST_OBJECT)) {
+            codegenParameter.dataType.equals(LanguageConventionResolver.LIST_OBJECT)) {
             codegenParameter.dataType = "array";
```

Comment:
Should this also be change to `ARRAY` from ApplicationConstants?

Reply:
We _could_. I was just focusing on `string` since it's what I needed for the change.